### DISCUSSION TITLE
Next option

### DISCUSSION THREAD

**Discussion by DamnYatin** [spencerwooo/onedrive-vercel-index#698] [Q&A] [answered]
I came here to request a thing that is option like from going from one path to another there are many photos that I wanna see but instead of single click I have to go to the back path and click on another picture and so on that it hurts a lot that's why I don't wanna do all these things
![1](https://user-images.githubusercontent.com/88388062/176721079-5b11d10b-bfff-457c-b1af-7cfefdaf8998.png)

Thanking if you put on this option.

**Comment by myl7** [comment, marked as answer]
You can use the bottom "List" on the right top of the page, and turn it to "Grid", to see previews of all pictures in the directory.
